hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMClientProtocolServer.java

```diff
@@ -675,17 +676,21 @@ public boolean startContainerBalancer(double threshold, int idleiterations,
     AUDIT.logWriteSuccess(buildAuditMessageForSuccess(
         SCMAction.START_CONTAINER_BALANCER, null));
     ContainerBalancerConfiguration cbc = new ContainerBalancerConfiguration();
+
     if (threshold > 0) {
       cbc.setThreshold(threshold);
     }
     if (maxDatanodesToBalance > 0) {
       cbc.setMaxDatanodesToBalance(maxDatanodesToBalance);
     }
     if (maxSizeToMove > 0) {
-      cbc.setMaxSizeToMove(maxSizeToMove);
+      cbc.setMaxSizeToMove(maxSizeToMove * OzoneConsts.GB);
```

Comment:
Can we also update the description for option maxSizeToMove? We can mention that for 10GB it should be mentioned as 10. I think it would be good to update the option name to maxSizeToMoveInGB.

Reply:
yea, this sounds good, will update this